Build error: internal/bootstrap.go:272:113: SA1018: calling strings.Replace with n == 0 will return no results, did you mean -1

```
openpaas on  main via 🐹 v1.18.7 on ☁️  (eu-west-2) 
❯ make
golangci-lint run ./...
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'structcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter.  Replaced by unused. 
WARN [linters_context] rowserrcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649. 
WARN [linters_context] sqlclosecheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649. 
WARN [linters_context] structcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649. 
internal/bootstrap.go:272:113: SA1018: calling strings.Replace with n == 0 will return no results, did you mean -1? (staticcheck)
		filepath.Join(baseDir, "nomad", "web.hcl"):         strings.Replace(nomadHealthCheck, "{DATACENTRE}", dcName, 0),
		                                                                                                              ^
make: *** [Makefile:3: lint] Error 1

openpaas on  main via 🐹 v1.18.7 on ☁️  (eu-west-2) took 6s 
❯ go version
go version go1.18.7 linux/amd64
```

--- a/internal/bootstrap.go
+++ b/internal/bootstrap.go
@@ -269,7 +269,7 @@ func makeConfigs(inventory Inventory, baseDir, dcName string) error {
 		filepath.Join(baseDir, "nomad", "nomad-server.service"): nomadServerService,
 		filepath.Join(baseDir, "nomad", "nomad-client.service"): nomadClientService,
 
-		filepath.Join(baseDir, "nomad", "web.hcl"):         strings.Replace(nomadHealthCheck, "{DATACENTRE}", dcName, 0),
+		filepath.Join(baseDir, "nomad", "web.hcl"):         strings.Replace(nomadHealthCheck, "{DATACENTRE}", dcName, -1),
 		filepath.Join(baseDir, "consul", "consul.service"): consulService,
 		filepath.Join(baseDir, "vault", "vault.service"):   vaultService,
 		filepath.Join(baseDir, "vault", "config.hcl"):      vaultConf,

